src/pages/RoomMembersPage.tsx

```diff
@@ -183,6 +183,7 @@ function RoomMembersPage({report, session, policies}: RoomMembersPageProps) {
                     return;
                 }
             }
+            const pendingChatMember = report?.pendingChatMembers?.find((member) => member.accountID === accountID.toString());
```

Comment:
Hi coming from https://github.com/Expensify/App/issues/38821 ✋

The issue is the deleted member offline didn't get strikethrough because we didn't get the last `pendingChatMembers`.